lib/web_console/testing/erb_helper.rb

```diff
@@ -0,0 +1,24 @@
+require 'web_console/testing/fake_middleware'
+
+# This class is to pre-compile 'templates/*.erb'.
+class ERBHelper
+  def initialize(path)
+    @erb  = ERB.new(File.read path)
+    @view = FakeMiddleware.new(
+      view_path: root('lib/web_console/templates'),
+    ).view
+  end
+
+  def build
+    @erb.result binding
```

Comment:
I'd also prefer to have parentheses when calling non DSL-ish methods. Again, to follow the rest of Rails' codebase.